ui/app/components/clients/running-total.hbs

```diff
@@ -9,44 +9,41 @@
     @description={{this.chartContainerText}}
     @timestamp={{@responseTimestamp}}
     @hasChartData={{true}}
-    data-test-chart="running total"
   >
     <:subTitle>
       <StatText
         @label="Running client total"
         @subText="The number of clients which interacted with Vault during this date range."
         @value={{@runningTotals.clients}}
         @size="l"
+        @tooltipText="This number is the total for the queried date range. The chart displays a monthly breakdown of total clients per month."
       />
     </:subTitle>
 
     <:stats>
       <div class="data-details-top has-top-padding-l">
         <div class="is-flex-row">
+          <StatText @label="Entity clients" @value={{@runningTotals.entity_clients}} @size="m" />
```

Comment:
These clients are related so I thought it made sense for them to be in the same row